exa/core/data.py

```diff
@@ -160,6 +162,38 @@ def _validate_data(self, df, reverse=False):
             raise TraitError(f"duplicates in {self.indexes}")
         return df
 
+    def load(self, name=None, directory=None):
+        name = name or self.name
+        directory = Path(directory) or exa.cfg.savedir
+        self.log.info(f"loading {directory / name}")
+        if (directory / f'{name}.yml').exists():
+            yml = self._from_yml(directory / f'{name}.yml')
+            for attr, vals in yml.items():
+                setattr(self, attr, vals)
+        if (directory / f'{name}.qet').exists():
+            self._data = pd.read_parquet(
+                directory / f'{name}.qet', columns=self.columns
+            )
+
+    def save(self, name=None, directory=None):
+        """Save the housed dataframe as a parquet file and related
+        metadata as a yml file with the same name."""
```

Comment:
Oh okay, I see this now - it answers one of the parts of my above comment ("exa" metadata + "standard" data saving).